### ISSUE TITLE
Ui component frontend : admin classes

### ISSUE THREAD

**Issue by CompactCodeEU** [magento/magento2#18801]
### Preconditions (*)

- Magento 2.2.6
- Mysql 5
- PHP 7.1
- Apache

### Summary (*)
When using some UI components it uses a template with admin classes.
There seems te be no difference in backend ui components and frontend ui components but they seem to be build for frontend ui components.

And sure we can implement custom templates and classes to mimic the frontend but this is not modular at all.

In respective to the already unclear documentation around the UI components this is truely something frustrating.

We try to use them as much as possible but it seems we bounce into a lot of limits causing ours of headache.

### Examples (*)

```
        <field name="infinite" sortOrder="11" formElement="checkbox">
            <argument name="data" xsi:type="array">
                <item name="config" xsi:type="array">
                    <item name="source" xsi:type="string">infinite</item>
                    <item name="default" xsi:type="number">1</item>
                </item>
            </argument>
            <settings>
                <dataType>boolean</dataType>
                <label translate="true">Unsubscribe after first mail</label>
            </settings>
            <formElements>
                <checkbox>
                    <settings>
                        <valueMap>
                            <map name="false" xsi:type="number">0</map>
                            <map name="true" xsi:type="number">1</map>
                        </valueMap>
                        <prefer>toggle</prefer>
                    </settings>
                </checkbox>
            </formElements>
        </field>
```

Results in a input without styling
![image](https://user-images.githubusercontent.com/41287340/47490098-49ab2d00-d848-11e8-891b-ca2d0560678e.png)

and with admin classes on the frontend

![image](https://user-images.githubusercontent.com/41287340/47490150-647da180-d848-11e8-9425-72d2c2eae2c9.png)

### Proposed solution

Provide frontend templates and a setting in the ui component where you can adjust the area.

**Comment by magento-engcom-team** [comment]
Hi @CompactCodeEU. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@CompactCodeEU do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by CompactCodeEU** [comment]
@magento-engcom-team give me 2.6-develop instance

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
Hi @CompactCodeEU Thank you for you report i'm closing this as duplicate to this -> https://github.com/magento/magento2/issues/15958.

**Comment by CompactCodeEU** [comment]
First of all when i opened an issue i had the option to report a bug or a developer expierence issue or a feature request.

I choose the developer experience issue. 
These have different required informations.
I have filled in the template
![image](https://user-images.githubusercontent.com/41287340/47490707-9e02dc80-d849-11e8-8ce8-5f3b303a37fe.png)

![image](https://user-images.githubusercontent.com/41287340/47490723-a65b1780-d849-11e8-976f-2659ef5cd3c8.png)

Not that i should point this out. This is your own github.........

Thanks for the duplicate. Glad it is solved in 2.3
